tests/core/reproducibility/test.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -eu -o pipefail
+
+function run_bazel() {
+    local output_base
```

Comment:
Assigning `"$1"` to another local here would improve readability.

Reply:
Done.